TAO/tao/Valuetype/ValueBase.h

```diff
@@ -360,13 +360,13 @@ namespace TAO_OBV_GIOP_Flags
   const CORBA::Long Value_tag_base    = 0x7fffff00L;
   const CORBA::Long Value_tag_sigbits = 0x7fffff00L;
   const CORBA::Long Codebase_url      = 1;
-  const CORBA::Long Type_info_sigbits = 0x00000006L;
+  const CORBA::Long Type_info_sigbits = 6;
```

Comment:
Maybe add the hex values as comment, very likely these come from the CORBA spec

Reply:
They are listed in Part 2 Section 9.3.4.8 but that's not meant to be an API in code directly (and these are in a TAO namespace, not CORBA).

I'll add a comment to the `-1` case -- I think it's clear that 6 is 0x6 and 8 is 0x8.

I'm also not sure why the two unsigned/signed conversions above (Value_tag_base and Value_tag_sigbits) didn't warn -- maybe they aren't used.